RELEASE/scripts/autoscend/quests/level_12.ash

```diff
@@ -644,6 +644,22 @@ boolean L12_preOutfit()
 		return false;
 	}
 
+	//use 1 wish if we can guarentee outfit drops via yellow ray
+	if(canGenieCombat() && auto_shouldUseWishes() && canYellowRay())
+	{
+		if(!get_property("auto_hippyInstead").to_boolean())
+		{
+			//want frat war outfit
+			auto_log_info("Trying to wish for a Orcish Frat Boy Spy, which we'll yellow ray for war outfit.");
+			return makeGenieCombat($monster[Orcish Frat Boy Spy]);
+		}
+		else
+		{
+			// don't waste resources speeding up slower route of fighting for hippies. 
+		}
+		
```

Comment:
```suggestion
		monster wishTarget = $monster[War Hippy Spy];
		if(!get_property("auto_hippyInstead").to_boolean())
		{
			wishTarget = $monster[Orcish Frat Boy Spy];
		}
		auto_log_info(`Trying to wish for a {wishTarget}, which we'll yellow ray for war outfit.`);
		return makeGenieCombat(wishTarget);		
```

Let's not just throw people who want to fight for the Hippies under the bus for no reason. People use autoscend to farm factoids in paths that have different war bosses.

Reply:
Good point. I'll make suggested updates. Why is ` string ` preferred over " string " for the log statement? Does that allow use of {variable}?

Reply:
Yeah with " for strings you have to concatenate your strings together manually e.g. "`Trying to wish for a " +  {wishTarget} + ", which..." while if you use ` you can insert function calls or variables directly into it wrapping them in {}. See https://wiki.kolmafia.us/index.php/Data_Types#string
I prefer it as it's easier to read in code and achieves the same result.